### DISCUSSION TITLE
Using replicate but it feels like my reference_file param isn't being trained on

### DISCUSSION THREAD

**Discussion by scoroman** [yl4579/StyleTTS2#183] [Q&A]
Hi, I've been using replicate to train my model on audio with 50 diffusion steps and it still sounds like the robotic default voice. Is my reference_file being recognized? Any help would be appreciated. 

output = replicate.run(
  model_id,
  input={
    "beta": 0.3,
    "reference_file": "training_audio.WAV", # also tried: "training_audio.mp4" "training_audio.mp3"

    "seed": 2,
    "text": text_to_synthesize,
    "alpha": 0.3,
    "diffusion_steps": 50,
    "embedding_scale": 5,
  }
)
print(output)

**Comment by yl4579** [comment]
You may want to follow up with https://github.com/yl4579/StyleTTS2/pull/97 or @alaradirik
